structuredefinitions/Extension-UKCore-SubscriptionHeaderVersion.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-UKCore-SubscriptionHeaderVersion" />
+  <url value="http://hl7.org/fhir/5.0/StructureDefinition/extension-Subscription.parameter" />
+  <version value="0.0.1" />
+  <name value="ExtensionUKCoreSubscriptionHeaderVersion" />
+  <title value="Extension UK Core Subscription Header Version" />
+  <status value="draft" />
+  <date value="2025-03-10" />
+  <publisher value="HL7 UK" />
+  <contact>
+    <name value="HL7 UK" />
+    <telecom>
+      <system value="email" />
+      <value value="ukcore@hl7.org.uk" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="An extension to replicate the changes within R5 to identify a parameter name and value for information passed to the channel for notifications. This is a backport of R5 Subscription.parameter element. In this context, the element is used to identify the headers related to API versions. E.g, Header1 or Header2." />
```

Comment:
I would remove references to header version in all fields, including the name and id as MNS has stated they plan to use the parameters for any additional information to be passed from the subscriber (not just versions)

Reply:
Will Extension-UKCore-SubscriptionAdditionalContent suffices

Reply:
Maybe Extension-UKCore-SubscriptionParameters? or SubscriptionAdditionalParameters?